build.gradle

```diff
@@ -189,9 +189,9 @@ tasks.register("verifyVersions") {
  * after the backport of the backcompat code is complete.
  */
 
-boolean bwc_tests_enabled = true
+boolean bwc_tests_enabled = false
 // place a PR link here when committing bwc changes:
-String bwc_tests_disabled_issue = ""
+String bwc_tests_disabled_issue = "https://github.com/elastic/elasticsearch/pull/71412"
```

Comment:
Where is the version logic in here needing bwc tests to be disabled?

Reply:
@rjernst It's an artifact of the builds we use in tests. We aren't using the full default distribution in REST tests. It means today these builds don't include the migrated roles by default. So the assertions in the 7.x REST tests don't assert the right set of roles in the right order for this change. That is, when we run the HTTP BWC tests from 7.x against the resulting distribution from this change, it's wrong. It will be fixed when this change is backported to 7.x.